mkt/constants/features.py

```diff
@@ -335,6 +336,12 @@
                              u'1GB RAM.'),
         'apis': (),
     }),
+    ('NFC', {
+        'name': _lazy(u'NFC'),
+        'description': _lazy(u'The app requires access to Near Field '
+                             u'Communication (NFC) API.'),
+        'apis': ('navigator.mozNfc',),
```

Comment:
Why is `mozNfc` not `mozNFC`? Esp since all the methods under it capitalize `NFC` (e.g. `getNFCPeer`, `getNFCTag`). Weird.